src/main/resources/google_checks.xml

```diff
@@ -210,8 +210,8 @@
         </module>
         <module name="AnnotationLocation">
             <property name="id" value="AnnotationLocationMostCases"/>
-            <property name="tokens"
-             value="CLASS_DEF, INTERFACE_DEF, ENUM_DEF, METHOD_DEF, CTOR_DEF"/>
+            <property name="tokens" value="CLASS_DEF, CTOR_DEF, ENUM_CONSTANT_DEF, ENUM_DEF"/>
+            <property name="tokens" value="INTERFACE_DEF, METHOD_DEF, PACKAGE_DEF"/>
```

Comment:
Since you are changing google_checks, please clarify which item in the google style guide says new tokens are ok.